models/truckModel.js

```diff
@@ -3,13 +3,18 @@
 // name your model 'Truck' (see the example in the slide deck for 'Book')
 // this is to match the name of the collection the data was imported into
 
-// var foodTruckSchema = new Schema({
-// 	name: String,
-// 	foodType: [String],
-// 	schedule: [String],
-// 	payment: [String],
-// 	description: String,
-// 	website: String,
-// 	Facebook: String,
-// 	Twitter: String
-// });
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+ var foodTruckSchema = new Schema({
+ 	name: {type: String},
+ 	foodType: {type: String},
+ 	schedule: {type: String},
+ 	payment: {type: String},
+ 	description: {type: String},
+ 	website: {type: String},
+ 	Facebook: {type: String},
+ 	Twitter: {type: String}
+ });
+
+module.exports = mongoose.model('Truck', truckModel);
```

Comment:
the second parameter to mongoose.model is the Schema just created, foodTruckSchema